sdk/confidentialledger/Azure.ResourceManager.ConfidentialLedger/tests/Scenario/CreateMccfTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ConfidentialLedger.Tests.Scenario
+{
+    [TestFixture("create")]
+    public class CreateMccfTest : MccfManagementTestBase
+    {
+        public CreateMccfTest(string testFixtureName) : base(true, RecordedTestMode.Playback, testFixtureName)
```

Comment:
minor: `RecoredTestMode.Playback` is the default value, could be removed

Reply:
removed. Thanks!